src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -188,6 +188,14 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISc
     /// <summary>
     /// Estimator for <see cref="IidChangePointDetector"/>
     /// </summary>
+    /// <p>Example code can be found by searching for <i>IidChangePointDetector</i> in <a href='https://github.com/dotnet/machinelearning'>ML.NET.</a></p>
+    /// <example>
+    /// <format type="text/markdown">
+    /// <![CDATA[
+    /// [!code-csharp[MF](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs)]
```

Comment:
>Dynamic/IidChangePointDetectorTransform.cs) [](start = 78, length = 43)

Looks great. Since you removed the copyright, the references on the other files will be skewed. Can you search for the other few references to the samples and just remove the range, like you did for Timeseries?

just search for "~/../docs/samples/docs/samples" there will be about 6 - 7 files.